chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1516,7 +1517,7 @@ impl Actor for PeerManagerActor {
 
             ctx.spawn(TcpListener::bind(server_addr).into_actor(self).then(
                 move |listener, act, ctx| {
-                    let listener = listener.unwrap();
+                    let listener = listener.with_context(||format!("Failed to PeerManagerActor at {}", server_addr)).unwrap();
```

Comment:
@matklad Is there a way to format errors nicely, with similar syntax like with `tracing`

```
error!(message = "Failed to PeerManagerActor, server_addr)
```